sbx/crossq/crossq.py

```diff
@@ -53,16 +53,15 @@ def __init__(
         self,
         policy,
         env: Union[GymEnv, str],
-        learning_rate: Union[float, Schedule] = 3e-4,
+        learning_rate: Union[float, Schedule] = 1e-3,
         qf_learning_rate: Optional[float] = None,
         buffer_size: int = 1_000_000,  # 1e6
-        learning_starts: int = 100,
+        learning_starts: int = 5_000,
```

Comment:
I would keep `100` to be consistent with the rest of SB3/SBX algorithms, see https://github.com/DLR-RM/stable-baselines3/pull/1785